raw_data_processor/src/main/scala/scalalab3/lyricsengine/reader/MSD.scala

```diff
@@ -0,0 +1,9 @@
+package scalalab3.lyricsengine.reader
+
+
+/**
+  * That class represent data from Million Song Dataset (MSD).
+  */
+case class MSD(words: Seq[String], songs: Seq[Song])
```

Comment:
Yes. Idea of this class (MDS), representation data from specific format, loaded by: http://labrosa.ee.columbia.edu
For other types of data we will write other handler. :)
I agree, we need use more informative naming.